oktopus/loss.py

```diff
@@ -35,15 +41,15 @@ def __call__(self, params):
         """Calls :func:`evaluate`"""
         return self.evaluate(params)
 
-    def fit(self, x0, method='Nelder-Mead', **kwargs):
+    def fit(self, optimizer='minimize', **kwargs):
         """
-        Minimizes the :func:`evaluate` function using :func:`scipy.optimize.minimize`
+        Minimizes the :func:`evaluate` function using :func:`scipy.optimize.minimize`,
+        :func:`scipy.optimize.differential_evolution`,
+        :func:`scipy.optimize.basinhopping`, or :func:`skopt.gp.gp_minimize`.
 
```

Comment:
Cool!   Somewhere in this docstring you will want to explicitely list the valid options of `optimizer`, and briefly expain which situation each optimizer is likely best-suited for (if you already have an understanding of that .... I bet it's complicated!)